charts/jenkins/templates/config.yaml

```diff
@@ -209,7 +209,14 @@ data:
     # Install missing plugins
     cp /var/jenkins_config/plugins.txt {{ .Values.master.jenkinsHome }};
     rm -rf {{ .Values.master.jenkinsRef }}/plugins/*.lock
-    /usr/local/bin/install-plugins.sh `echo $(cat {{ .Values.master.jenkinsHome }}/plugins.txt)`;
+    version () { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+
+
+    if [ -n "$(command -v jenkins-plugin-cli)" 2>/dev/null ] && [ $(version $(jenkins-plugin-cli --version)) -ge $(version "2.1.1") ]; then
+      jenkins-plugin-cli -w /usr/share/jenkins/jenkins.war --verbose -f {{ .Values.master.jenkinsHome }}/plugins.txt;
```

Comment:
Can we use the long argument —war instead? I think this is the default so it shouldn’t actually be needed too

Reply:
> Can we use the long argument —war instead? 

Done

> I think this is the default so it shouldn’t actually be needed too

Agreed, except the ref directory is configurable, which i missed. so I fix.